backend/FwLite/FwLiteProjectSync.Tests/EntrySyncTests.cs

```diff
@@ -191,4 +204,34 @@ public async Task CanInsertComplexFormComponentViaSync(bool componentThenComplex
             .Excluding(c => c.Order)
             .Excluding(c => c.Id));
     }
+
+    [Fact]
+    public async Task CanSyncNewEntryReferencedByExistingEntry()
+    {
+        // arrange
+        // - before
+        var existingEntryBefore = await Api.CreateEntry(new() { LexemeForm = { { "en", "existing-component" } } });
+
+        // - after
+        var existingEntryAfter = existingEntryBefore.Copy();
+        var newEntry = new Entry() { Id = Guid.NewGuid(), LexemeForm = { { "en", "complex-form" } } };
+        var newComplexFormComponent = ComplexFormComponent.FromEntries(newEntry, existingEntryAfter);
+        existingEntryAfter.ComplexForms.Add(newComplexFormComponent);
+        newEntry.Components.Add(newComplexFormComponent);
+
+        // act
+        await EntrySync.Sync([existingEntryBefore], [existingEntryAfter, newEntry], Api);
+
+        // assert
+        var actualExistingEntry = await Api.GetEntry(existingEntryAfter.Id);
+        actualExistingEntry.Should().BeEquivalentTo(existingEntryAfter, options => options
+            .For(e => e.ComplexForms).Exclude(c => c.Id)
+            .For(e => e.ComplexForms).Exclude(c => c.Order));
```

Comment:
remind me why we exclude order? do we just not map that at all? I don't see why we couldn't, it wouldn't be hard.

Reply:
In this case it's, because in the CRDT case the `existingEntryAfter` doesn't have the "correct" order value.
And it doesn't, because it doesn't need to, because it's an internal implementation detail.

The bigger reason is perhaps, because the FwData api never uses/sets the Order. That wouldn't fail the test in this case I don't think, but consumers of our API (e.g. our tests) can't rely on Order, because the different implementations don't behave the same way. We could have the FwData API set the order frorm 1 to N when things leave the API, but that still wouldn't be the same as the Crdt API, which could have gaps and decimals.